app/assets/javascripts/pageflow/editor/views/theme_item_view.js

```diff
@@ -0,0 +1,54 @@
+/** @api private */
+pageflow.ThemeItemView = Backbone.Marionette.ItemView.extend({
+  tagName: 'li',
+  template: 'pageflow/editor/templates/theme_item',
+
+  mixins: [pageflow.selectableView],
+
+  selectionAttribute: 'theme',
+
+  ui: {
+    themeName: '.theme_name',
+    useButton: '.use_theme',
+    inUseRegion: '.theme_in_use'
+  },
+
+  events: {
+    'click .use_theme': function(event) {
+      this.configuration.set('theme_name', this.model.get('name'));
+      pageflow.updateStylesheet('theme');
+    },
+    'mouseenter': function() {
+      this.select();
+    },
+    'mouseleave': function() {
+      this.onClose();
```

Comment:
Calling the `onClose` lifecycle handler here feels a bit unidiomatic. The fact that `selectableView` implements `onClose` is more of an implementation detail. Even more, I am wondering if this is necessary at all. I think it would be just fine if the most recently hovered theme stayed selected even after the mouse left the item. This would also be more in line with a touch friendly interface: We could, for example, bind both `mouseenter` and `click` to ensure themes can be selected on platforms without mouse events.

The only thing we would need to add is some visual hint (like a changed background color) indicating which theme is currently selected.